updateHostsFile.py

```diff
@@ -21,15 +21,16 @@
 import time
 from glob import glob
 
-import lxml  # noqa: F401
-from bs4 import BeautifulSoup
+try:
+    import requests
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError("This project's dependencies have changed. The Requests library ("
+                              "https://requests.readthedocs.io/en/master/) is now required.")
```

Comment:
This feels a little verbose, but that's fine. 👍

Reply:
How about "The Requests library (https://requests.readthedocs.io/en/master/) is now a required dependency." ? Is "a required dependency" a bit awkward?